docs/modules/ROOT/pages/about/changelog.adoc

```diff
@@ -1,6 +1,6 @@
 = Changelog
 
-An extensive changelog is available https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md[here].
+An extensive changelog is available https://github.com/rubocop/rubocop/releases[here].
```

Comment:
As this page is dedicated to the changelog, I don't think we should change this particular link. But feel free to expand the section if you feel that something is missing - e.g. add a warning about the size of the changelog and an encouragement to use the release notes instead.

Reply:
How about linking to https://github.com/rubocop/rubocop/tree/master/relnotes?

Or, how do you feel about linking to the raw file where github doesn't try to render it?

Reply:
I'd be fine with that.